gems/codders-dataset/OSVDB-108582.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Hrm. I think the distinction to be made is more that we should have a copy per flaw-people-should-be-aware-of. MITRE treating both mysql and postgres flaws as the *same thing*, from the user's perspective, is correct imho. 

I think we should be *exhaustive* in terms of having sources for things, but meh on having exact copies from upstream.